src/applications/vaos/utils/events.js

```diff
@@ -45,3 +45,45 @@ export function recordItemsRetrieved(type, count) {
   });
   resetDataLayer();
 }
+
+export const NULL_STATE_FIELD = {
+  TYPE_OF_CARE: 'type-of-care',
+};
+
+/**
+ * Records events for appointment details null states
+ *
+ * @export
+ * @param {Object} nullStates This is the dictionary containing the null state
+ *   details to be logged. The keys are the keys of NULL_STATE_TYPE and the
+ *   values are booleans indicating whether that field is missing in the
+ *   appointment details (i.e. true for missing, false for not missing). If an
+ *   information type is not applicable for the appointment type (e.g. Provider
+ *   for Claim Exam) do not include an entry for the field in the dictionary.
+ */
+export function recordAppointmentDetailsNullStates(nullStates) {
```

Comment:
I'm not actually certain that this is the best spot to put this utility. Any suggestions?